src/plugins/UploadArtifact/UploadArtifact.js

```diff
@@ -69,16 +66,6 @@ define([
         }
     };
 
-    UploadArtifact.prototype.transfer = async function (hash, storage, name) {
-        const filename = `${this.projectId}/artifacts/${name}`;
-        const gmeStorageClient = await Storage.getBackend('gme').getClient(this.logger);
-        const dataInfo = gmeStorageClient.createDataInfo(hash);
-        const content = await gmeStorageClient.getFile(dataInfo);
-
-        const {id, config} = storage;
-        const dstStorage = await Storage.getBackend(id).getClient(this.logger, config);
-        return await dstStorage.putFile(filename, content);
-    };
 
     UploadArtifact.prototype.getAssetNameFromHash = async function(hash){
```

Comment:
One Question on this is should we remove any `dot` extensions from the file?

Reply:
I would probably leave the file as-is.